docs/_config.yml

```diff
@@ -14,7 +14,7 @@ ALLUXIO_MASTER_VERSION_SHORT: master
 ALLUXIO_MAJOR_VERSION: master
 # We must inline the version string (e.g., "1.4.0-SNAPSHOT") rather than using the macro of Alluxio version.
 # Otherwise the macro name remains in the output.
-ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/client/alluxio-1.8.0-SNAPSHOT-client.jar
+ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/client/alluxio-1.9.0-SNAPSHOT-client.jar
```

Comment:
update line 9 to `1.8.0` too?

Reply:
I will update it when 1.8.0 is released and available for download